docs/setup/connect-to-elasticsearch.asciidoc

```diff
@@ -1,53 +1,67 @@
 [[connect-to-elasticsearch]]
-== Add your data
-++++
-<titleabbrev>Add data</titleabbrev>
-++++
+== Add an integration
 
-To start working with your data in {kib}, use one of the many ingest options,
-available from the home page.
-You can collect data from an app or service,
-or upload a file that contains your data.  If you're not ready to use your own data,
-add a sample data set and give {kib} a test drive.
+The best way to add data to the Elastic Stack is to use one of our many integrations.
+Alternatively, you can add a sample data set or upload a file.
+
+[float]
+[[add-an-integration]]
+=== Integrations
+
+Integrations are a way to stream in logs, metrics, traces,
+content, and more from your apps, endpoints, infrastructure, cloud, network,
+workplace tools, and other common sources in your ecosystem. All
+integrations are available in a single view, and
+{kib} guides you there from the Welcome screen and home page. Start
+with one of the three promoted integrations, browse the categories, or search for a specific integration.
 
 [role="screenshot"]
-image::images/addData_home_7.15.0.png[Built-in options for adding data to Kibana:  Add data, Add Elastic Agent, Upload a file]
+image::images/add-integration.png[Integrations page from which you can choose integrations to start collecting and analyzing data]
+
+If an integration is available for both
+https://www.elastic.co/guide/en/fleet/master/beats-agent-comparison.html[Elastic Agent and Beats], the *Integrations* view defaults to the
+Elastic Agent integration, if it is generally available (GA).
+To switch to a
+Beats integration, use the filter on the left.
+
+[role="screenshot"]
+image::images/integration-filter.png[Filter for Elasic Agent or Beats in Integrations, width=50%]
```

Comment:
```suggestion
image::images/integration-filter.png["Filter for Elasic Agent or Beats in Integrations", width=50%]
```

Reply:
I tested this locally and the quotation marks make no difference to the output.